package.json

```diff
@@ -30,45 +30,44 @@
   "lint-staged": {
     "*.js": [
       "prettier --write",
-      "eslint",
-      "git add"
+      "eslint"
     ]
   },
   "dependencies": {
-    "ajv": "6.12.3",
-    "caseless": "0.12.0",
-    "clone": "2.1.2",
-    "commander": "3.0.2",
+    "ajv": "6.12.6",
+    "commander": "7.1.0",
     "content-type": "1.0.4",
     "curl-trace-parser": "0.0.10",
-    "deep-equal": "1.1.1",
+    "deep-equal": "2.0.5",
     "http-string-parser": "0.0.6",
-    "json-pointer": "0.6.0",
     "media-typer": "1.1.0",
-    "tv4": "1.3.0",
-    "url": "0.11.0"
+    "tv4": "1.3.0"
   },
   "devDependencies": {
-    "@rollup/plugin-json": "4.0.1",
-    "chai": "4.2.0",
-    "cross-spawn": "7.0.1",
+    "@rollup/plugin-json": "4.1.0",
+    "caseless": "0.12.0",
```

Comment:
Why are we moving `caseless` into `devDependencies`? It's used on runtime:

https://github.com/apiaryio/gavel.js/blob/22ccd2ddb659fe252cff55ea3de74938db557d5a/lib/utils/tv4-to-headers-message.js#L14

The same is true for the `json-pointer`:

https://github.com/apiaryio/gavel.js/blob/22ccd2ddb659fe252cff55ea3de74938db557d5a/lib/utils/to-gavel-result.js#L19

That's also the reason the CI fails at the moment.

Reply:
There's a related discussion in https://github.com/apiaryio/gavel.js/pull/537

Reply:
@artem-zakharchenko both these packages are relevant only to tests:
- file `test/unit/utils/tv4-to-headers-message.test.js` for `caseless`
- file `test/unit/utils/to-gavel-result.test.js` for `json-pointer`

both can be moved into devDeps.
I'll move these utility files into test folder and that should solve the failing `lint` part of tests, ok?